ui/src/overridden/backoffice/Importer/ImporterTaskDetails/ReportDetails.js

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Item, Icon, Table, Message } from 'semantic-ui-react';
+import _isEmpty from 'lodash/isEmpty';
+import { Link } from 'react-router-dom';
+import { BackOfficeRoutes } from '@inveniosoftware/react-invenio-app-ils';
+
+const displayRecordLink = (id, label) => (
+  <Link to={id} target="_blank">
+    {label}
+  </Link>
+);
+
+const displayValue = (value, urlGenerator, getId, getTitle) => {
+  return _isEmpty(value)
+    ? ''
+    : displayRecordLink(urlGenerator(getId(value)), getTitle(value));
+};
+
+const displayValues = (values, urlGenerator, getId, getTitle) => {
+  return _isEmpty(values)
+    ? ''
+    : values.map((value, index) => (
+        <span key={index}>
+          {index > 0 ? ', ' : ''}
+          {displayValue(value, urlGenerator, getId, getTitle)}
+        </span>
+      ));
+};
+
+export class ReportDetails extends React.Component {
+  renderErrorMessage = message => {
+    let title;
+    let content;
+    const split = message.split(': ');
+    if (split.length > 1 && split[0] === 'LossyConversion') {
```

Comment:
the error message normally should be provided by the backend, instead of custom parsing it's text on the frontend

Reply:
Currently we format the errors that way so they can be both human readable and parsable without ambiguity. I'm not sure if we would gain much by structuring the data (in json for instance), data which consists of just two fields.